index.html

```diff
@@ -0,0 +1,646 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/resources/css/my-index.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="icon" href="https://auroville.org/images/av_favi_icon-bb57269cdce6407f4a931a9065e58035.png?vsn=d">
+    <title>Welcome to Auroville | Auroville</title>
+  </head>
+  <body>
+  <!--Main Content Starts Here-->
+  <!--Navigation Bar Starts-->
+<div class="navbar">
```

Comment:
Use semantic elements and make your code more readable. for example intead of writing <div> tag for the navbar you can use <nav> tag,  for footer  you can use <footer> tag. you can also seperate the pages, section by section by using <section> tag and then inside it you can use <div>. don't use div tag for everything.